src/schedule/zephyr_ll.c

```diff
@@ -253,7 +254,8 @@ static void zephyr_ll_run(void *data)
 				break;
 			default:
 				/* reschedule */
-				task->start = sch->ll_domain->next_tick;
+				task->start = sch->ll_domain->next_tick +
+					sch->ll_domain->ticks_per_ms * pdata->period / 1000;
```

Comment:
Hmm, something's odd here. Normally this should be set as per the timer tick rate in zephyr_domain_timer_fn(). Is this not happening in your case?

Reply:
@lenghonglin can it be that your `CONFIG_SYS_CLOCK_TICKS_PER_SEC` is wrong?

Reply:
> Hmm, something's odd here. Normally this should be set as per the timer tick rate in zephyr_domain_timer_fn(). Is this not happening in your case?
`domain_is_pending` will compare the `stask->start` with `k_uptime_ticks()` , 

see this part

https://github.com/thesofproject/sof/blob/f58896fa146db525b6d33742319af2e302765083/src/schedule/zephyr_ll.c#L205

Reply:
> @lenghonglin can it be that your `CONFIG_SYS_CLOCK_TICKS_PER_SEC` is wrong?

Maybe nope.....We can refer to the logic of `task->start` calculation in `ll_schedule.c`

see https://github.com/thesofproject/sof/blob/f58896fa146db525b6d33742319af2e302765083/src/schedule/ll_schedule.c#L138